devenv/prometheus.yml

```
@@ -0,0 +1,7 @@
+global:
```

YAMLlint (1.37.1)

[error] wrong new line character: expected \n

(new-lines)